srcpkgs/gramps/template

```diff
@@ -1,7 +1,7 @@
 # Template file for 'gramps'
 pkgname=gramps
-version=5.1.4
-revision=2
+version=5.1.5
+revision=1
 build_style=python3-module
 hostmakedepends="pkg-config intltool python3"
```

Comment:
```suggestion
hostmakedepends="pkg-config gettext python3"
```
and test if it generates the same files

Reply:
Well it works, but at first glance for some reason it doesn't generate a desktop file and a few other files:

```zsh
/usr/share/applications/gramps.desktop
/usr/share/metainfo/gramps.appdata.xml
/usr/share/mime/packages/gramps.xml
/usr/share/mime-info/gramps.keys
```

Also I'm not able to test other languages in depth but I ran it like so:

```zsh
env LC_ALL=nl_BE.UTF-8 LANGUAGE="sv" gramps
```

And it seems to work fine.

Reply:
You are right, in that case we shouldn't switch to plain gettext for now.